### DISCUSSION TITLE
Missing "Lathe"?

### DISCUSSION THREAD

**Discussion by zzador** [Bforartists/Bforartists#3513] [Ideas]
Hello everybody. I tried BForArtists a couple of days ago and was instantly hyped. With my little knowledge about 3D Modelling/Animation that I gathered in the past 30 years I was able the instantly get things done and just with mouse without knowing any shortcut. For me as a hobbyist this is a great deal as it feels that this type of 3D Animation packages is long gone since the 90ies.
I did a very simple Cornell-Box were I created a box, flipped the normals, deleted one of the faces and put some primitives inside. Nothing big.

Today I wanted to created something to animate by physics. A small animation loop where a metal ball bearing fall into a funnel and spins down. To make the funnel I created a bezier spline and closed it and then I wanted to apply an "Lathe"-operation to the spline. I searched for appox. one hour and was very depressed at that point that I would have to consult a tutorial nonetheless. I found one pretty fast on youtube and the solution made me baffeled. I did look in the Modifier-Section for a Lathe-Modifier but found none. Even more surprising that the modifier is called "Screw" but does exactly what I wanted. Please think about it with regards to making Blender more usable/accessable to hobbyists and artists. People got used to names like "lathe" in the past 30 years. Why change?

**Comment by ReinerBforartists** [comment]
Hi zzador, 

I will turn this into a discussion since it is nothing that we can really solve.

It's the Blender names for things. Yes, it can be frustrating to find not the tools that you expect from other software. But we are in a dilemma here. We cannot rename the Blender tools since then all the Blender tutorials would become worthless for Bforartists. I hope you understand this.

We have a very helpful and active Discord server in case you have further questions :)

https://discord.com/invite/yKuR77v

Kind regards
Reiner
